operations/helm/charts/mimir-distributed/templates/memcached/_memcached-statefulset.tpl

```diff
@@ -58,6 +58,10 @@ spec:
         - name: {{ . }}
       {{- end }}
       {{- end }}
+      {{- if .extraVolumes }}
+      volumes:
+      {{ toYaml .extraVolumes | indent 8}}
```

Comment:
Doesn't render correctly for me with indent, instead of nindent:
```
   volumes:
              - name: extra-volume
          secret:
            secretName: myminiosecret
```

```suggestion
        {{- toYaml .extraVolumes | nindent 8}}
```